agent/scripts/newrelic.ini.private.template

```diff
@@ -193,3 +193,14 @@
 ;          hooks instrumented.
 ;
 ;newrelic.framework.wordpress.hooks_skip_filename=""
+
+; Setting: newrelic.distributed_tracing.pad_trace_id
+; Type   : bool
+; Scope  : system
```

Comment:
This says that the new, INI setting has the ‘SYSTEM’ scope: https://github.com/newrelic/newrelic-php-agent/blob/48316ddef44ffb3378a91771adc8fc513ef37bb1/agent/scripts/newrelic.ini.private.template#L199 However the implementation says otherwise: https://github.com/newrelic/newrelic-php-agent/blob/48316ddef44ffb3378a91771adc8fc513ef37bb1/agent/php_nrini.c#L3084 I don’t think this is correct - there shouldn’t be a discrepancy like that.

Reply:
Fixed in [f40d8da](https://github.com/newrelic/newrelic-php-agent/pull/929/commits/f40d8da6d36061dab021aab96655c85cf3dd6025)